software.html

```diff
@@ -29,28 +29,21 @@ <h2 id="reference">Reference implementation</h2>
 
     <h2 id="browsers">Browsers</h2>
 
-    <p>Currently there are no browsers that have native FLIF support. We hope for this to change at some point,
-        but browser developers are understandably hesitant to support new image formats.
+    <p>Currently there are no browsers that have native FLIF support. The situation is unlikely to change, though,
+        since FLIF has been obsoleted by <a href="https://jpeg.org/jpegxl">JPEG XL</a>.
         There is however a javascript based implementation:</p>
 
     <ul>
         <li><strong><a href="https://uprootlabs.github.io/poly-flif">Poly-FLIF</a></strong> - A Polyfill for browsers that don't
             natively support FLIF.</li>
     </ul>
 
-    <p>If you want to encourage browser developers to add FLIF support, maybe these pointers might help:</p>
-
-    <ul>
-        <li><a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1240692"><strong>Mozilla</strong>: BugZilla Bug 1240692 - (FLIF) Implement support for Free Lossless Image Format (FLIF)</a></li>
-        <li><a href="https://bugs.chromium.org/p/chromium/issues/detail?id=539120"><strong>Chromium</strong>: Issue 539120 – Feature-Request: FLIF-Support</a></li>
-    </ul>
```

Comment:
It might be worth keeping these links, for historical reference, for example embedded in the "unlikely to change" sentence above.